### ISSUE TITLE
Oneway arrows on roads (highway tags) are slow to zoom out

### ISSUE THREAD

**Issue by Altonss** [organicmaps/organicmaps#3598]
On OM version 2022.09.22-3-FDroid on Android, the oneway arrows are very slow (maybe the slowest) to update when zooming out.

**Comment by biodranik** [comment]
Do you mean that they appear after everything else is drawn?

What is your device model and OS version?

**Comment by Altonss** [comment]
> Do you mean that they appear after everything else is drawn?

Yes exactly! They are still very much zoomed while the rest is redrawn more quickly.



> What is your device model and OS version?

Android 13, GrapheneOS on Pixel 4a

**Comment by Jean-BaptisteC** [comment]
@Altonss can you retry on the latest version?
